tests/Unit/Evolution/Systems/Cce/Test_LinearSolve.cpp

```diff
@@ -294,7 +294,8 @@ template <typename BondiValueTag, typename Generator>
 void test_pole_integration_with_linear_operator(
     const gsl::not_null<Generator*> gen, size_t number_of_radial_grid_points,
     size_t l_max) noexcept {
-  UniformCustomDistribution<double> dist(0.1, 5.0);
+  // the relevant linear solves usually involve fairly small wave amplitudes
```

Comment:
As a non CCE expert, I'm not sure what you mean by "relevant". I suggest (assuming I correctly understood) something like "The linear solves used in CCE usually involve...", or any improved version of this that you like.